### ISSUE TITLE
Resh problem in Jerusalem

### ISSUE THREAD

**Issue by BillMeyerRSA** [charlesLoder/hebrew-transliteration#39]
Getting odd result with transliteration
This word "יְרוּשָׁלִַם" from this sentence: כִּי הִנְנִי קֹרֵא, לְכָל-מִשְׁפְּחוֹת מַמְלְכוֹת צָפוֹנָה--נְאֻם-יְהוָה; וּבָאוּ וְנָתְנוּ אִישׁ כִּסְאוֹ פֶּתַח שַׁעֲרֵי יְרוּשָׁלִַם, וְעַל כָּל-חוֹמֹתֶיהָ סָבִיב, וְעַל, כָּל-עָרֵי יְהוּדָה.

![image](https://user-images.githubusercontent.com/38781014/207861735-180dcb88-d2ba-4020-9087-69ac28eb2a80.png)

**Comment by johnlockejrr** [comment]
Very weird, the R gets deleted when Qamets appears after Shin:
`יְרוּשׁ -> yǝrûš`
`יְרוּשָׁ -> yǝûšā`

**Comment by charlesLoder** [comment]
Interestingly, this is actually an issue with the site, and not the package. As far as I can tell, the package itself produces the correct results.

I'll look into this more

**Comment by johnlockejrr** [comment]
Just checked the package, same problem.

```
$ node t.js
yǝûšālaim
```

**Comment by charlesLoder** [comment]
Ok, I figured it out.

In the syllabification package, havarotjs, there is a [`strict` option](https://github.com/charlesLoder/havarotjs/blob/49e9b487ce68d01107ddb7c84ceeff38a942e9e5/src/text.ts#L143-L162) designed to allow people to input poorly pointed text.

The transliteration package defaults to `false`, however, so people can use it with fewer errors.

On `main` of havarotjs, this issue is fixed. I just haven't created a release for it yet.

I'll try to do that over the weekend

**Comment by johnlockejrr** [comment]
Yes you are right!

```
john@vps:~/work/hebrew-transliteration$ tail -n 2 dist/schemas/tiberianKhan.js
    allowNoNiqqud: false,
    strict: true
};

john@vps:~/hebrew-transliteration$ node t.js
yǝrûšālaim
```

**Comment by BillMeyerRSA** [comment]
> Interestingly, this is actually an issue with the site, and not the package. As far as I can tell, the package itself produces the correct results.
> 
> I'll look into this more

It is in the package. I wrote a JavaScript program to batch transliterate a number of words and picked up the issue there. Then double checked on the website and confirmed.

**Comment by charlesLoder** [comment]
Fixed in [v2.3.1](https://github.com/charlesLoder/hebrew-transliteration/blob/main/CHANGELOG.md).

The site is updated now as well
